### PR TITLE
Add summary

### PR DESCRIPTION
**Pull request by nmercadeb**

### CODE REVIEW

File: R/appStatic.R [hunk 1]
```diff
@@ -22,6 +25,12 @@ exportStaticApp <- function(result = emptySummarisedResult(),
   omopgenerics::assertLogical(open, length = 1)
   omopgenerics::assertCharacter(logo, length = 1, null = TRUE)
   omopgenerics::assertCharacter(title, length = 1)
+  omopgenerics::assertLogical(summary, length = 1)
+  if (summary) {
```
Comment (catalamarti) [inline R/appStatic.R:29]: can we have this in an internal function? Then if we expand is easier to deal with it?

File: tests/testthat/test-appStatic.R [hunk 2]
```diff
@@ -69,6 +69,13 @@ test_that("CohortCharacteristics shiny", {
   expect_snapshot(serverStatic(resultTypes = names(getChoices(result))) |> cat(sep = "\n"))
   unlink(paste0(tdir, "/shiny/"), recursive = TRUE)
 
+  # use summary
+  expect_no_error(exportStaticApp(result = result, directory = tdir, summary = TRUE))
+  expect_true("shiny" %in% list.files(tdir))
+  expect_snapshot(uiStatic(choices = getChoices(result), summary = capture.output(summary(result), type = "message"), logo = NULL) |> cat(sep = "\n"))
+  expect_snapshot(uiStatic(choices = getChoices(result), summary = capture.output(summary(result), type = "message"), logo = "HDS") |> cat(sep = "\n"))
+  unlink(paste0(tdir, "/shiny/"), recursive = TRUE)
```
Comment (catalamarti) [inline tests/testthat/test-appStatic.R:77]: there is no tests with summary = FALSE

**Review by catalamarti — CHANGES_REQUESTED** [review]

**Comment by nmercadeb** [comment]
@catalamarti  ready to merge (probabilty will be conflicts between both prs and test snapshots)

**Comment by nmercadeb** [comment]
@catalamarti ready

**Pull request merged**